lightning/src/ln/chan_utils.rs

```diff
@@ -660,13 +661,24 @@ pub fn make_funding_redeemscript(broadcaster: &PublicKey, countersignatory: &Pub
 	let broadcaster_funding_key = broadcaster.serialize();
 	let countersignatory_funding_key = countersignatory.serialize();
 
+	make_funding_redeemscript_from_slices(&broadcaster_funding_key, &countersignatory_funding_key)
+}
+
+pub(crate) fn make_funding_redeemscript_from_node_ids(broadcaster: &NodeId, countersignatory: &NodeId) -> Script {
+	let broadcaster_funding_key = broadcaster.as_slice();
+	let countersignatory_funding_key = countersignatory.as_slice();
+
+	make_funding_redeemscript_from_slices(broadcaster_funding_key, countersignatory_funding_key)
+}
```

Comment:
Hmm was this thin wrapper necessary compared to casting to slices and just using the  `..._from_slices` in other places? Just curious if it's just ergonomics, but otherwise looks fine!

Reply:
Oh you're right that makes more sense